src/App.js

```diff
@@ -1,16 +1,13 @@
-import './App.css';
-import { AddMovie } from './components/AddMovie';
-import logo from './assets/images/netflix-logo.png';
-import { SearchBanner } from './components/Banners/SearchBanner/index';
+import "./App.css";
```

Comment:
The import order is not 100% correct here.

Reply:
resolved